### ISSUE TITLE
New integration test framework fails with Galley being nil when running in kube env. 

### ISSUE THREAD

**Issue by pitlv2109** [istio/istio#13352]
Related to this PR https://github.com/istio/istio/pull/13115. When I ran existing tests, e.g `sds_citadel_flow_test.go`, the test failed with error `galley must not be nil`. It ran fine before https://github.com/istio/istio/pull/13115 was merged.

```
g := galley.NewOrFail(t, ctx, galley.Config{})
	p := pilot.NewOrFail(t, ctx, pilot.Config{
		Galley: g,
	})
	appInst := apps.NewOrFail(t, ctx, apps.Config{Pilot: p})
```

**Comment by howardjohn** [comment]
Yeah the tests fail in prow anyways which is why they were missed. simple fix, just add `Galley: g` to apps.Config

**Comment by pitlv2109** [comment]
Yes. I have included the fix in #13353. Will close this issue once that PR has been checked in.
